packages/teams-js/test/public/chat.spec.ts

```diff
@@ -101,6 +113,17 @@ describe('chat', () => {
       return expect(chat.openGroupChat(chatRequest)).rejects.toThrowError('The library has not yet been initialized');
     });
 
+    it('should throw error if it is not supported in runtime config', async () => {
```

Comment:
Maybe specify the capability so it's like 

`it('should throw error if chat capability is not supported in runtime config'`?

Reply:
Same for all the other test descriptions

Reply:
Resolved in the latest commit